scripts/deploy_subscription_manager.py

```diff
@@ -30,5 +32,6 @@ def main(account_name : str = 'test'):
         calldata,
         {'from': deployer})
 
-    subscription_manager = Contract.from_abi("SubscriptionManager", transparent_proxy.address, subscription_manager_logic.abi, owner=None)
+    subscription_manager = Contract.from_abi(
+        "SubscriptionManager", transparent_proxy.address, subscription_manager_logic.abi, owner=None)
```

Comment:
Controversial opinion:  Linters make some lines more ugly and less readable to follow arbitrary line length rules.  This is an example of that.  Nonetheless this PR does look good to me.  Only suggestion I can make is potentially to just adjust the maximum line length.  whats the average screen resolution in 2022 anyways? 😄

Reply:
We tend to use black (https://pypi.org/project/black/) everywhere anyway. So if people have `pre-commit` it should catch things before they hit this linter